provider-ui/components/Footer.js

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react'
```

Comment:
I might be missing something obvious. Why was this footer rewritten?

Reply:
Footer wasn't rewritten, and it was moved to its own component. the `_app.js` was bloated with styles that weren't being used

Reply:
Ok, I see. Thanks for this. Makes much sense. 👍